unlock-protocol-com/blog/subgraph-v2.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Unlock Protocol Subgraph Upgrade with The Graph
+subTitle: An upgraded version of the subgraph is now available
+publishedDate: September 29, 2022
```

Comment:
please change metadata tag to **publishDate** (instead of publishedDate). **publishDate** is correct and will enable blog post to render in blog. static site generator doesn't recognize "publishedDate."

Reply:
@clemsos